osu.Game/Overlays/Toolbar/Toolbar.cs

```diff
@@ -19,65 +19,25 @@ namespace osu.Game.Overlays.Toolbar
 {
     public class Toolbar : OverlayContainer
     {
-        private const float height = 50;
+        public const float HEIGHT = 50;
 
         public Action OnHome;
         public Action<PlayMode> OnPlayModeChange;
 
         private ToolbarModeSelector modeSelector;
-        private Box solidBackground;
-        private Box gradientBackground;
 
         private const int transition_time = 250;
 
         private const float alpha_hovering = 0.8f;
         private const float alpha_normal = 0.6f;
 
-
-        protected override void PopIn()
-        {
-            MoveToY(0, transition_time, EasingTypes.OutQuint);
-            FadeIn(transition_time, EasingTypes.OutQuint);
-        }
-
-        protected override void PopOut()
-        {
-            MoveToY(-DrawSize.Y, transition_time, EasingTypes.InQuint);
-            FadeOut(transition_time, EasingTypes.InQuint);
-        }
-
-        protected override bool OnHover(InputState state)
-        {
-            solidBackground.FadeTo(alpha_hovering, transition_time, EasingTypes.OutQuint);
-            gradientBackground.FadeIn(transition_time, EasingTypes.OutQuint);
-            return true;
-        }
-
-        protected override void OnHoverLost(InputState state)
-        {
-            solidBackground.FadeTo(alpha_normal, transition_time, EasingTypes.OutQuint);
-            gradientBackground.FadeOut(transition_time, EasingTypes.OutQuint);
-        }
+        public override bool Contains(Vector2 screenSpacePos) => true;

```

Comment:
It'd make the intention explicit in case someone looks at the code. Additionally, input won't be piped through all the input handling methods of the toolbar _all the time_. If we'd do that too much it might even become a performance concern at some point.